### ISSUE TITLE
feature request: kitty @ --to

### ISSUE THREAD

**Issue by jbrea** [jpalardy/vim-slime#234]
Hi

I love vim-slime, thank a lot!
Currently I use it with kitty.
Recently I realised that there is a second way to do [remote control in kitty](https://sw.kovidgoyal.net/kitty/remote-control.html#tutorial).
To try it, I changed the KittySend function to
```
  call system("kitty @ --to unix:/tmp/" . shellescape(a:config["window_id"]) .
    \  " send-text --from-file " . g:slime_paste_file)
```
This works. But what is the best way to incorporate both ways of using kitty remote control into vim-slime?

**Comment by jpalardy** [comment]
hi @jbrea,

This is similar (though not exactly the same) to the current code.

What's the advantage of this approach? (or of supporting both?)

**Comment by jbrea** [comment]
With the current approach messaging works only when run inside a kitty window.
With the approach I proposed above one can control kitty also from programs/scripts not running inside a kitty window. But this increased flexibility comes at the cost of having to start kitty like this
`kitty -o allow_remote_control=yes --listen-on unix:/tmp/mykitty`.

My initial motivation for the new approach was to send text to a kitty window that runs in a different OS window. But now I realized that one can detach kitty windows into new OS windows. So I am fine with the current approach of vim-slime.

I am closing this for now, if you don't mind.
